R/calculate_grade.R

```diff
@@ -14,4 +14,41 @@
 #' @examples calculate_grade('/DSCI524.csv')
 calculate_grade <- function(input_file_path) {
 
+    cwd <- getwd()
+    path <- paste0(cwd, "/", input_file_path)
+    course_info <- read.csv(path, header = TRUE)
+    # course_info <- read.csv(input_file_path, col.names = TRUE, header = FALSE)
+
+    print(course_info)
+
+    error_msg_missing_value <- "Course component grades is missing for "
+    error_msg_format <- "Course component grades with incorrect format (not 2 decimal places) for "
+    grade_msg <- "Course garde is "
+    percent_sign <- "%"
+    course_grade <- 0
+    return_msg <- ""
+
+    # Check if all course components have garde present
```

Comment:
Typo for "grade"

Reply:
I will update :(

Reply:
@shlrley the typo is now fixed with a new commit and push here again